### PR TITLE
[wrap] Add and enable libpng

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
Originally:
 - c74e968991c663f23f67118050630b8c4e684670
 - b09db64585cf5234227470caf1a9dd1789d2df11
 - 5bafbf139134a5952a6780dfb6a48dce14d8dce8
 - 6001e0a3534af31a0387459170146daf96cb631c

### CODE REVIEW

File: src/lib/evas/meson.build [hunk 1]
```diff
@@ -4,11 +4,9 @@
...
 #
 
-if not sys_windows
-  png = dependency('libpng')
-else
-  png = declare_dependency()
-endif
+png = dependency('libpng',
+  fallback: ['libpng', 'png_dep']
+)
```
Comment (Coquinho) [inline src/lib/evas/meson.build:9]: I think there is no need to use 3 lines here:
```suggestion
png = dependency('libpng', fallback: ['libpng', 'png_dep'])
```

File: .gitignore [hunk 2]
```diff
@@ -101,4 +101,6 @@ env.bat
 
 # Subprojects
 subprojects/*
+!subprojects/packagefiles
+!subprojects/packagefiles/*
```
Comment (Coquinho) [inline .gitignore:105]: Why ignoring package files if this PR addsit?

Comment (joaoantoniocardoso) [reply .gitignore:105]: My bad, that shouldn't be committed.

Comment (Coquinho) [reply .gitignore]: ohhhhh, right right, I didn't note the `!`, thanks.

**Pull request merged**